### PR TITLE
require_realm: update schema, require realm on PUT/POST

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: applications/crossbar/priv/couchdb/schemas/connectivity.json [hunk 1]
```diff
@@ -43,10 +44,12 @@
                     "type": "integer"
                 }
             },
+            "required": true,
             "type": "object"
         },
         "name": {
             "description": "Human-friendly name of the trunkstore account",
+            "required": true,
```
Comment (fenollp) [inline applications/crossbar/priv/couchdb/schemas/connectivity.json]: why require name? it says "require realm"

Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/connectivity.json]: Name is required too

Comment (fenollp) [inline applications/crossbar/priv/couchdb/schemas/connectivity.json]: Your commit message was a lie!

**Comment by fenollp** [comment]
@jamesaimonetti fwd port this plz

**Comment by jamesaimonetti** [comment]
KAZOO-4687 will be the proper fixes for 4.x. This was a 3.22-only patch

**Pull request merged**